apps/web/src/locales/en/messages.json

```
@@ -2213,6 +2213,9 @@
   "disable": {
     "message": "Turn off"
   },
+  "orgUserDetailsNotFound": {
+    "message": "Org user details not found"
+  },
```

We refer to orgUsers as Members in our UI copy:

```suggestion
  "orgUserDetailsNotFound": {
    "message": "Member details not found."
  },
```